packages/plasma-new-hope/src/components/Slider/ui/Handler/Handler.styles.ts

```diff
@@ -19,27 +19,31 @@ export const StyledValue = styled.span<{ isLargeThumb?: boolean }>`
     line-height: var(${tokens.valueLineHeight});
 `;
 
-export const HandlerStyled = styled.div<{ isLarge?: boolean }>`
+export const HandlerStyled = styled.div<{ isLarge?: boolean; isPointerHidden?: boolean }>`
+    --thumb-size: ${({ isLarge }) => (isLarge ? `var(${tokens.thumbSizeLarge})` : `var(${tokens.thumbSize})`)};
+
     cursor: pointer;
     position: absolute;
     z-index: 1;
     top: 0;
-    left: 0;
+    left: calc(var(--thumb-size) / -2 - 0.0625rem);
     bottom: 0;
     display: flex;
     align-items: center;
     justify-content: center;
 
-    --thumb-size: ${({ isLarge }) => (isLarge ? `var(${tokens.thumbSizeLarge})` : `var(${tokens.thumbSize})`)};
-
     &.${classes.verticalOrientation} {
         right: 0;
+        left: 0;
+        top: calc(var(--thumb-size) / -2 - 0.0625rem);
         bottom: auto;
+        margin-left: -0.0625rem;
         align-items: flex-start;
         justify-content: center;
 
         ${StyledValue} {
-            margin: 0;
+            margin: ${({ isPointerHidden }) => (isPointerHidden ? 'calc(var(--thumb-size) / 2 )' : 'calc(-0.125rem)')} 0
```

Comment:
_minor_

Тут лишний `calc`?
```suggestion
            margin: ${({ isPointerHidden }) => (isPointerHidden ? 'calc(var(--thumb-size) / 2 )' : '-0.125rem')} 0
```